### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by chichaparadise**
https://chichaparadise.github.io/front/

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+
+.productSize{
+    padding: 5px 20px;
+    border: 1px solid black;
+    margin-right: 10px;
+    cursor: pointer;
+    font-size: 20px;
+}
+
+.product__button{
```
Comment (katetushkan) [inline css/style.css:342]: smth went wrong with naming...

File: login.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            <div class="navbarTop">
+                <div class="navbarTopItem">
+                    <img src="img/logo.png" alt="logo" class="navbarTopItem__logo">
+                </div>
+                <div class="navbarTopItem">
+                    <a class="navbarTopItem__login" href="index.html">Main Page</a>
+                </div>
+            </div>
+        </nav>
+        <div class="auth">
```
Comment (katetushkan) [inline login.html:22]: maybe `section`?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                </div>
+                <div class="productSizes">
+                    <div class="productSize">42</div>
+                    <div class="productSize">43</div>
+                    <div class="productSize">44</div>
+                </div>
+                <button class="product__button">BUY NOW</button>
+            </div>
+            <div class="payment">
+                <div class="personalInfo">
```
Comment (katetushkan) [inline index.html:121]: section maybe?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+            <article class="feature">
+                <img class="feature__img" src="img/contact.png" alt="">
+                <span class="feature__title">CONTACT US!</span>
+                <span class="feature__description">Keep In Touch Via Email And Support System</span>
+            </article>
+        </section>
+        <section class="product">
+            <img src="img/nitejogger.png" alt="" class="product__img">
+            <div class="productDetails">
+                <h1 class="productDetails__title">NITE JOGGER</h1>
```
Comment (katetushkan) [inline index.html:104]: only one `h1` an be on page

File: index.html [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+                <div class="navbarTopItem">
+                    <select class="currency">
+                        <option value="USD" class="currency__item" selected>USD</option>
+                        <option value="UER" class="currency__item">EUR</option>
+                    </select>
+                    <a class="navbarTopItem__login" href="login.html">Auth</a>
+                </div>
+            </div>
+            <div class="menu">
+                <h3 class="menu__item">Adidas</h3>
```
Comment (katetushkan) [inline index.html:36]: `h3` only applicable for section's or document's headers

**Review by katetushkan — CHANGES_REQUESTED** [review]

File: css/auth.css [hunk 6]
```diff
@@ -0,0 +1,96 @@
+@import url("../css/style.css");
+
+.auth {
+    margin-top: 10%;
```
[the remark this answers is not in the document]
Comment (chichaparadise) [reply css/auth.css]: fixed

File: login.html (hunk 2 above)
Comment (chichaparadise) [reply login.html]: changed to `section`

File: index.html [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+                    <li class="footerList__item">
+                        <a href="#">Feedback</a>
+                    </li>
+                    <li class="footerList__item">
+                        <a href="#">Stories</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="footerMenu">
+                <h3 class="footerMenu__title">Products</h1>
```
[the remark this answers is not in the document]
Comment (chichaparadise) [reply index.html]: substituted with `<p>`

File: index.html (hunk 3 above)
Comment (chichaparadise) [reply index.html:121]: replaced with `article`

File: index.html (hunk 4 above)
Comment (chichaparadise) [reply index.html]: regardless there are 5 of h1 tags, these are slider items, so there will be only one h1 element per page

File: index.html (hunk 5 above)
Comment (chichaparadise) [reply index.html]: replaced with `<p>`

File: index.html [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+                        <img src="img/search.png" width="20" height="20" alt="" class="search__icon">
+                    </div>
+                </div>
+                <div class="navbarTopItem">
+                    <select class="currency">
+                        <option value="USD" class="currency__item" selected>USD</option>
+                        <option value="UER" class="currency__item">EUR</option>
+                    </select>
+                    <a class="navbarTopItem__login" href="login.html">Auth</a>
+                </div>
```
[the remark this answers is not in the document]
Comment (chichaparadise) [reply index.html]: fixed

**Comment by chichaparadise** [comment]
followed your tips and fixed issues, hope you will be satisfied)))

File: index.html [hunk 9]
```diff
@@ -101,8 +101,8 @@ <h2 class="sliderItem__price">$120</h2>
         <section class="product">
             <img src="img/nitejogger.png" alt="" class="product__img">
             <div class="productDetails">
-                <h1 class="productDetails__title">NITE JOGGER</h1>
-                <h2 class="productDetails__price">$149</h2>
+                <h2 class="productDetails__title">NITE JOGGER</h2>
```
Comment (katetushkan) [inline index.html:104]: See, There is a rule, so the page/section can has only one header. Also, u can't use lower headers without higher ones. For example, u can't use `h2` without `h1`. Or use `h1` and `h3` with skipping `h2`. If u use cards and use `<h...>` tags in header so u can still stick to the rules of the ``article`'s area and start with the high order heading.

**Review by katetushkan — CHANGES_REQUESTED** [review]

**Review by katetushkan — CHANGES_REQUESTED** [review]
Mobile version ?

**Comment by chichaparadise** [comment]
what's wrong with is?

**Comment by chichaparadise** [comment]
Oh, I've recently discovered that responce differs in inspection mode and original browsing, I'll fix it A$AP

**Pull request merged**